library/Zend/InputFilter/InputFilterPluginManager.php

```diff
@@ -46,6 +46,7 @@ public function __construct(ConfigInterface $configuration = null)
         parent::__construct($configuration);
 
         $this->addInitializer(array($this, 'populateFactory'));
+        $this->addAbstractFactory('Zend\InputFilter\InputFilterAbstractServiceFactory');
```

Comment:
Well, I'll remove it. I added the abstract factory because I thought it would make the feature adoption easier. If it is not activated by default, I bother it won't be used. But maybe I'm wrong.